cpp/include/Ice/Endpoint.h

```diff
@@ -116,32 +94,6 @@ namespace Ice
     {
     public:
         IPEndpointInfo() = default;
-
-        /**
-         * One-shot constructor to initialize all data members.
-         * @param underlying The information of the underlying endpoint or null if there's no underlying endpoint.
-         * @param timeout The timeout for the endpoint in milliseconds.
-         * @param compress Specifies whether or not compression should be used if available when using this endpoint.
-         * @param host The host or address configured with the endpoint.
-         * @param port The port number.
-         * @param sourceAddress The source IP address.
-         */
-        IPEndpointInfo(
```

Comment:
The code coverage report showed that all these one-shot ctors weren't called, with the exception of the OpaqueEndpointInfo ctor.